src/app/components/WorkDetail/WorkDetail.jsx

```diff
@@ -61,20 +63,26 @@ class WorkDetail extends React.Component {
     return (
       <main id="mainContent">
         <div className="nypl-full-width-wrapper">
-          <div className="nypl-page-header">
-            <Breadcrumbs
-              links={[
-                {
-                  href: `/search?q=${this.props.searchQuery}&field=${this.props.searchField}`,
-                  text: 'Search Results',
-                },
-                {
-                  href: `/work?workId=${work.uuid}`,
-                  text: 'Work Details',
-                },
-              ]}
-              pageType="details"
-              onClickHandler={handleReset}
+          <div className="search-wrapper">
+            <div className="nypl-page-header">
```

Comment:
There's no `nypl-page-header` in the search container component. Is this meant to have different html and style?

Reply:
Good point. Removed